src/javascripts/components/Chat.js

```diff
@@ -1,19 +1,130 @@
-var React = require('react');
-var ChatBox_settings = require('./ChatBox_settings');
-var ChatBox = require('./ChatBox');
-var ChatInput = require('./ChatInput');
-
+import React from 'react'
+import ChatBoxSettings from './ChatBoxSettings'
+import ChatBox from './ChatBox'
+import ChatInput from './ChatInput'
+import { resMsgs } from '../helper'
 
 class Chat extends React.Component {
+  constructor() {
+      super()
+      this.state = {
+        messages: [
+          {
+            user: "blue",
+            message: "blah blah blah"
+          }, 
+          {
+            user: "yellow",
+            message: "Lorem ipsum dipsum"
+          },
+          {
+            user: "yellow",
+            message: "Hallo lorem"
+          }
+        ]
+      }
+  }  
+
+  componentWillMount() {
+    const localStorageRef = localStorage.getItem(`messages`)  
+    if(localStorageRef) {
+      this.setState({
+        messages: JSON.parse(localStorageRef)
+      })
+    }
+  }
+
+  componentDidMount() {
+    const chatbox = document.querySelector('.chatbox')
+    chatbox.scrollTo(0, chatbox.scrollHeight)
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const chatbox = document.querySelector('.chatbox')
+    chatbox.scrollTo(0, chatbox.scrollHeight)
+  }
```

Comment:
Same here -- move into `ChatBox` and use a `ref`.  Also, you're doing the same thing twice, so seems like you could abstract this into a shared method.